lib/services/jira.rb

```diff
@@ -378,12 +383,16 @@ def update_issue(issue_info, resource, initiative, version, parent)
 
     if @feature == resource
       # Only update custom_fields and due dates for features, not requirements.
-      # This will cause an issue with two-way field syncing if we constantly overwrite requirements' custom fields
+      # This will cause an issue with two-way field syncing if we constantly
+      # overwrite requirements' custom fields
       issue.fields
         .merge!(mapped_custom_fields(@feature, issue_type))
```

Comment:
It doesn't do exactly the same though, but I'm not sure of the difference or why this was like this in the first place.. If resource is a requirement this line runs the custom field mapping against the feature.